engines/bops_admin/app/controllers/bops_admin/application_types_controller.rb

```diff
@@ -12,6 +13,8 @@ def index
     end
 
     def show
+      @application_type_requirements = ApplicationTypeRequirement.where(application_type_id: params[:id])
```

Comment:
I think this isn't necessary because `set_application_type_requirements` will do it, except for the mismatched variable name.